src/navigationDrawer/navigationDrawer.component.tsx

```diff
@@ -75,8 +81,16 @@ class NavigationDrawer extends Component<CombinedNavigationProps> {
         id={`plugin-link-${plugin.link.replace(/\//g, '-')}`}
         button
       >
+        <img
+          className={this.props.classes.menuLogo}
+          alt={plugin.logoAltText}
+          src={
+            plugin.src &&
+            plugin.src +
+              (this.props.darkMode ? plugin.logoDarkMode : plugin.logoLightMode)
+          }
+        />
```

Comment:
We should probably make this `img` tag conditional on the plugin actually providing the logo options, otherwise we'll get an empty image tag (which isn't particularly a problem, just a little strange).

Reply:
The reason I left it in was so the alt text (i.e. DataGateway) would display if the image failed to load (though obviously if neither `src` nor `alt` is set then it's worthless). But to be honest the formatting on that wasn't great anyway, it'd probably be better to include that in the `<ListItemText>` so it's consistent with the rest of the display name in the event we don't have the image? Either way I'll make the <img/> conditional like you say